infor-design/enterprise
# 2975 - Add a new setting for opacity in Modal/Message

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR adds a feature to allow user to control the opacity of the modal/message using `overlayOpacity` setting.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/2975

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

**Test 1: Modal Component**
- Navigate to http://localhost:4000/components/modal/test-overlay-opacity.html
- Click `Show Modal` button
- Modal overlay opacity level should be `0.3` (To verify, inspect the element)

**Test 2: Message Component**
- Navigate to http://localhost:4000/components/message/test-overlay-opacity
- Test the different messages with different level of opacity (indicated on button)
- Modal overlay opacity level should match the indicated label of the button

**Included in this Pull Request**:
